docs/release_docs/Uat-ci-cd_ReleaseFLow.md

```diff
@@ -0,0 +1,33 @@
+## UAT-CI-CD
+
+There are two ways to use or trigger uat-CI-CD workflow
+1. By creating a Pull Request to merge into MAIN branch
+2. Whenever a Branch with prefix of 'release' create a pull request
+![uat-ci-cd_Workflow](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/uat-ci-cd_workflow.png?raw=true)
+![uat](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/Uat-ci-cd_example.png?raw=true)
+
+### JOBS
+
+#### CodeScan-ESLint
```

Comment:
Can we keep the explanation for this section the same across all docs? I see it is being used in a couple of places.